src/components/Match/matchColumns.jsx

```diff
@@ -51,6 +52,13 @@ export const heroTdColumn = {
   sortFn: true,
 };
 
+export const heroTdChatColumn = {
```

Comment:
Note this is still babel stage 3: https://babeljs.io/docs/plugins/preset-stage-3/

It won't work in Node, you have to do Object.assign() instead